lib/smart_answer/calculators/benefit_cap_calculator_data_query.rb

```diff
@@ -1,12 +1,15 @@
 module SmartAnswer::Calculators
   class BenefitCapCalculatorDataQuery
-    attr_reader :data, :benefits, :questions
     def data
       @data ||= YAML.load_file(Rails.root.join('lib', 'data', 'benefit_cap_data.yml'))
     end
 
+    def rates
+      @rates ||= data.fetch("rates")
+    end
+
     def benefits
-      @benefits ||=  @data.fetch("benefits").with_indifferent_access
```

Comment:
Minor: I prefer to rewrite history to avoid these small problems appearing in the first place.